modules/end-user-guide/pages/automatic-token-injection.adoc

```diff
@@ -0,0 +1,21 @@
+:navtitle: Automatic {orch-name} token injection
+:keywords: user-guide, token, injection
+:page-aliases: .:automatic-token-injection, overview:automatic-token-injection
+
+[id="automatic-token-injection_{context}"]
+= Automatic {orch-name} token injection
+
+This section describes how to use the {orch-name} user token that is automatically injected into workspace containers.
+
+.Procedure
+
+. Open the {prod-short} dashboard and start a workspace.
+
+. Once the workspace is started open a terminal in the container that contains {prod-short} CLI.
+
+. Execute {prod-short} CLI commands which allow you to run commands against {orch-name} cluster. CLI can be used for deploying applications, inspecting and managing cluster resources, and viewing logs. {orch-name} user token will be used during the execution of the commands. 
+
++
+image::integration/token-injection.png[Token Injection in IDE,link="../_images/integration/token-injection.png"]
+
+WARNING: The automatic token injection functionality currently works only on the OpenShift infrastructure.
```

Comment:
"function" is easier than "functionality" and provides the same content.

Reply:
I decided to drop `functionality` wording since it is redundant